code/game/machinery/machinery.dm

```diff
@@ -464,10 +464,8 @@ Class Procs:
 /obj/machinery/tesla_act(var/power)
 	..()
 	if(prob(85))
-		emp_act(2)
+		explosion(src.loc,1,2,4,flame_range = 2)
```

Comment:
Way to make the tesla go from something that creates practically no lag to something that will.

I realize that with all the deferring, explosions are much better, but this is pretty ridiculous---and noisey